Add a page for media related to notebooks for all project (#71)
Iota-School/notebooks-for-all: tests/media.md

# Notebooks for all media

A collection of multimedia references to Notebooks for All.

## Articles 

> * [Making Jupyter Notebooks More Accessible]

Reviewer: I'm not sure which one this is and couldn't find it on my own. I'd rather have a link for this or remove it if we can't verify.

Reviewer: Nevermind, it was at the bottom of the page. My mistake! 
```suggestion
* [Notebooks For All: Making Jupyter Notebooks More Accessible - Astrobites]( https://astrobites.org/2023/06/10/notebooks-for-all/)
```

> * [Making Jupyter Notebooks More Accessible]

Reviewer: ```suggestion
* [Notebooks For All: Making Jupyter Notebooks More Accessible - Astrobites]
```

> * referenced in [Data Driven Understanding of Data
> Science Notebook Inaccessibility]

Reviewer: ```suggestion
* referenced in [Notably Inaccessible -- Data Driven Understanding of Data
Science Notebook Inaccessibility](https://arxiv.org/abs/2308.03241)
```

Reviewer: Correcting to match Markdown reference formatting but keep more accurate title.

```suggestion
* referenced in [Notably Inaccessible -- Data Driven Understanding of Data
Science Notebook Inaccessibility]
```

> * [SciPy 2023](https://gihyo.jp/article/2023/08/scipy2023) written in Japanese

Reviewer: ```suggestion
* Referenced in [SciPy 2023 Conference Report - gihoyo.jp](https://gihyo.jp/article/2023/08/scipy2023)
```

Reviewer: Correcting to match formatting but keep more accurate title.
```suggestion
* Referenced in [SciPy 2023 Conference Report - gihoyo.jp](https://gihyo.jp/article/2023/08/scipy2023)
```

## Videos

> * [Patrick Smyth, Jenn Kotler Notebooks for All: Accessibility & Jupyter Notebooks | JupyterCon 2023](https://www.youtube.com/watch?v=LJMS9R-KFXQ)

Reviewer: Changing format for consistency. 

```suggestion
* [Patrick Smyth, Jenn Kotler Notebooks for All: Accessibility & Jupyter Notebooks - JupyterCon 2023](https://www.youtube.com/watch?v=LJMS9R-KFXQ)
```

* [Basic Jupyter Notebook Accessibility](https://www.youtube.com/watch?v=KsUF_HjA97U&t=318s)

> * [arXiv Accessibility Forum 2023 — Dr. Patrick Smyth: Hackccessibility for All](https://www.youtube.com/watch?v=1v3jFCWQfI8)

Reviewer: Changing format for consistency.

```suggestion
* [ Dr. Patrick Smyth: Hackccessibility for All - arXiv Accessibility Forum 2023 ](https://www.youtube.com/watch?v=1v3jFCWQfI8)
```

* [STScI Day of Accessibility playlist](https://www.youtube.com/playlist?list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1)   

>     * [Introduction to Day of Accessibility]
>     * [Notebooks for All]
>     * [Illuminating the Universe with Accessible Text Descriptions]
>     * [Joshua Miele - Day of Accessibility Keynote]
>     * [Hearing the Light - Astronomy Data Sonification]
>     * [Learn to Use a Screen Reader]

Reviewer: ```suggestion
    * [Introduction to Day of Accessibility](https://www.youtube.com/watch?v=9ELaUM1fRNc&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=1&pp=iAQB)
    * [Notebooks for All](https://www.youtube.com/watch?v=Cmvh_nZoqM8&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=2&pp=iAQB)
    * [Illuminating the Universe with Accessible Text Descriptions](https://www.youtube.com/watch?v=Xpka7pDl2fI&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=3&pp=iAQB)
    * [Joshua Miele - Day of Accessibility Keynote](https://www.youtube.com/watch?v=pR29v4-J6i0&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=4&pp=iAQB)
    * [Hearing the Light - Astronomy Data Sonification](https://www.youtube.com/watch?v=-MBpGKoULyc&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=5&pp=iAQB)
    * [Learn to Use a Screen Reader](https://www.youtube.com/watch?v=F189lurxSbs&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=6&pp=iAQB)
```

> * [Accessibility best practices for authoring Jupyter notebooks] @ JupyterCon

Reviewer: Since I see another JupyterCon talk on this list and this is the name of the SciPy talk, I think this is what is meant.

```suggestion
* [Accessibility best practices for authoring Jupyter notebooks - SciPy 2023](https://youtu.be/onjRLjveGxk)
```

Reviewer: Correcting to match Markdown reference formatting but keep more accurate title.

```suggestion
* [Accessibility best practices for authoring Jupyter notebooks - SciPy 2023]
```

> [Introduction to Day of Accessibility]: https://www.youtube.com/watch?v=9ELaUM1fRNc&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=1&t=9s&pp=iAQB
> [Notebooks for All]: https://www.youtube.com/watch?v=Cmvh_nZoqM8&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=2&pp=iAQB
> [Illuminating the Universe with Accessible Text Descriptions]: https://www.youtube.com/watch?v=Xpka7pDl2fI&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=3&pp=iAQB
> [Joshua Miele - Day of Accessibility Keynote]: https://www.youtube.com/watch?v=pR29v4-J6i0&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=4&t=6s&pp=iAQB
> [Hearing the Light - Astronomy Data Sonification]: https://www.youtube.com/watch?v=-MBpGKoULyc&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=5&t=10s&pp=iAQB
> [Learn to Use a Screen Reader]: https://www.youtube.com/watch?v=F189lurxSbs&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=6&pp=iAQB
> [Making Jupyter Notebooks More Accessible]: https://astrobites.org/2023/06/10/notebooks-for-all/
> [Data Driven Understanding of Data Science Notebook Inaccessibility]: https://arxiv.org/pdf/2308.03241.pdf

Reviewer: I would remove these since they are repeated above.

```suggestion
```

> [Accessibility best practices for authoring Jupyter notebooks]: https://www.youtube.com/watch?v=onjRLjveGxk

Reviewer: I would remove these since they are repeated above.

```suggestion
```

> [Introduction to Day of Accessibility]: https://www.youtube.com/watch?v=9ELaUM1fRNc&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=1&t=9s&pp=iAQB
> [Notebooks for All]: https://www.youtube.com/watch?v=Cmvh_nZoqM8&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=2&pp=iAQB
> [Illuminating the Universe with Accessible Text Descriptions]: https://www.youtube.com/watch?v=Xpka7pDl2fI&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=3&pp=iAQB
> [Joshua Miele - Day of Accessibility Keynote]: https://www.youtube.com/watch?v=pR29v4-J6i0&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=4&t=6s&pp=iAQB
> [Hearing the Light - Astronomy Data Sonification]: https://www.youtube.com/watch?v=-MBpGKoULyc&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=5&t=10s&pp=iAQB
> [Learn to Use a Screen Reader]: https://www.youtube.com/watch?v=F189lurxSbs&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=6&pp=iAQB
> [Making Jupyter Notebooks More Accessible]: https://astrobites.org/2023/06/10/notebooks-for-all/
> [Data Driven Understanding of Data Science Notebook Inaccessibility]: https://arxiv.org/pdf/2308.03241.pdf
> 
> [Accessibility best practices for authoring Jupyter notebooks]: https://www.youtube.com/watch?v=onjRLjveGxk

Reviewer: Updating the references to match changed titles in other comments.

```suggestion
[Introduction to Day of Accessibility]: https://www.youtube.com/watch?v=9ELaUM1fRNc&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=1&t=9s&pp=iAQB
[Notebooks for All]: https://www.youtube.com/watch?v=Cmvh_nZoqM8&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=2&pp=iAQB
[Illuminating the Universe with Accessible Text Descriptions]: https://www.youtube.com/watch?v=Xpka7pDl2fI&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=3&pp=iAQB
[Joshua Miele - Day of Accessibility Keynote]: https://www.youtube.com/watch?v=pR29v4-J6i0&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=4&t=6s&pp=iAQB
[Hearing the Light - Astronomy Data Sonification]: https://www.youtube.com/watch?v=-MBpGKoULyc&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=5&t=10s&pp=iAQB
[Learn to Use a Screen Reader]: https://www.youtube.com/watch?v=F189lurxSbs&list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1&index=6&pp=iAQB
[Notebooks For All: Making Jupyter Notebooks More Accessible - Astrobites]: https://astrobites.org/2023/06/10/notebooks-for-all/
[Notably Inaccessible -- Data Driven Understanding of Data
Science Notebook Inaccessibility]: https://arxiv.org/pdf/2308.03241.pdf

[Accessibility best practices for authoring Jupyter notebooks - SciPy 2023]: https://www.youtube.com/watch?v=onjRLjveGxk
```
